src/app/shared/post-it-collections/post-it-collections.component.ts

```diff
@@ -238,15 +238,35 @@ export class PostItCollectionsComponent implements OnInit, AfterViewInit, OnDest
   }
 
   public createCollection(collection: CollectionModel) {
-    this.store.dispatch(new CollectionsAction.Create({collection}));
+    this.store.dispatch(new CollectionsAction.Create({collection, callback: this.onCreateCollection()}));
+  }
+
+  private onCreateCollection(): (collection: CollectionModel) => void {
+    const query = this.query;
+    const store = this.store;
+    return collection => {
+      if (queryIsNotEmpty(query)) {
+        store.dispatch(new NavigationAction.AddCollectionToQuery({collectionId: collection.id}));
```

Comment:
I think we should not add collection to the query even if the `collectionIds` array in the query is empty because it means that the user wants to see all collections.

Imagine you have only fulltext in the query, add a new collection and suddenly all other collections will disappear since the query was not empty and you added only the newly created collection to it.